csrc/scheduler/matmul_utils.cpp

```diff
@@ -115,6 +115,8 @@ inline c10::optional<MmaOptions::MacroType> getMmaOp(
       return (use_small_n) ? MacroType::Turing_16_8_16
                            : MacroType::Turing_16_16_16;
     case 80:
+    case 86:
+    case 89:
```

Comment:
Without the above two lines, the following tests will fail on RTX 4090 and RTX 3090:
```C++
NVFuserTest.FusionMatmulSegmenterBasicMatmulStrictCheckTT_CUDA
NVFuserTest.FusionMatmulSegmenterBasicMatmulRelaxedCheck_CUDA
```

With the above two lines, RTX 3090 is no longer failing, but I don't know if this is correct for RTX 4090 because I am still seeing `NVFuserTest.FusionMatmulSegmenterBasicMatmulStrictCheckTT_CUDA` failing with number checks:
```C++
Validation error in output 0 on line 3198 in file /home/gaoxiang/Fuser/test/test_gpu_tensorcore.cpp.
  Detected abs error of: 0.0308151
    absolute tolerance was set to 3.3451e-05
    and relative tolerance set to 3.3451e-07
```

Can Ada GPU use Ampere macros?

Reply:
Just checked and Ada supports the same set of macros as Ampere, so this is correct.

The difference in results may come from arch differences in Tensor Cores. How about removing this very strict test, as it is very specific and results are stable only for small set of problem shapes on some subset of GPUs in the arch?

Reply:
Thanks for checking! I will update this test to run only on 80 and 86.

Reply:
I have updated the test, now it should be passing/skipped.